x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -259,6 +259,7 @@ export const getManagementFilteredLinks = async (
   // may see failed HTTP requests in the browser console. This is the reason that
   // `hasKibanaPrivilege()` is used below.
   if (
+    currentUser &&
```

Comment:
This and the one below prevent having UI error when currentUser is empty and there are no roles (`roles` array is undefined). That was causing an error when doing `currentUser.roles.includes(...)...`.